collections/_posts/2022-11-23-Multi_Channel_Stochastic_Variational_Inference.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: review
+title: "Multi-Channel Stochastic Variational Inference for the Joint Analysis of Heterogeneous Biomedical Data in Alzheimer's Disease"
+tags: fusion multimodality machine-Learning
+author: "Romain Deleat-Besson"
+cite:
+    authors: "Luigi Antelmi, Nicholas Ayache, Philippe Robert and Marco Lorenzi"
+    title:   "Multi-Channel Stochastic Variational Inference for the Joint Analysis of Heterogeneous Biomedical Data in Alzheimer's Disease"
+    venue:   "Machine Learning in Clinical Neuroimaging (MLCN), MICCAI Workshop 2018"
+pdf: "https://arxiv.org/pdf/1808.03662.pdf"
+---
+
+
+
+# Notes
+
+* Unsupervised method.
+* **Gaussian linear case** (for validation purposes)
+* Another paper based on deep learning methods was done in 2019 by the same authors: [Sparse Multi-Channel Variational Autoencoder](http://proceedings.mlr.press/v97/antelmi19a/antelmi19a.pdf).
+* A [Github repository](https://gitlab.inria.fr/epione_ML/mcvae) is available for the 2019 paper.
+
+* Link to the [VAE tutorial](https://creatis-myriad.github.io/tutorials/2022-09-12-tutorial-vae.html) 
+
+
+# Highlights
+
+* The aim of this paper is to take into account several types of inputs (which are referred to as _channels_ in the paper) for a better joint analysis.
+* They proposed a novel multi-channel stochastic generative model.
+* Their proposed method can have applications for generic data fusion techniques.
+
+
+# Introduction
+
+When a patient suffers from Alzheimer’s disease, physicians use lots of sources of information. However, simple univariate correlation analysis are limited in modeling power. To overcome the limitations of mass-univariate analysis, methods such as PLS, RRR, CCA, and their variants (non-linearity, multi-channels) were used in biomedical research. The main issue is that those methods are not generative.
+The authors proposed a novel multi-channel stochastic generative model for the joint analysis of multi-channel heterogeneous data.
+
+
+# Methods
+
+Let $$ \textbf{x}  = \{\textbf{x}_c\}_{c=1}^{C} $$ be a single observation of a set of C channels, where each $$ \textbf{x}_c \in \mathbb{R}^{d_c} $$ is a $$ d_c $$-dimensional vector.
+
+* $$ p(\textbf{z}) $$ : prior
+* $$ p(\textbf{x}_c \vert \textbf{z}, \pmb{θ}_c) $$ : likelihood distribution
```

Comment:
As much as possible, I would avoid using special unicode characters (the ŷ in this case) if a basic latex function can do the trick (\hat in thi case). It seems to work here, but I'm worried about edge-cases/updates that could possibly not support special characters.:
```suggestion
* $$ p(\textbf{x}_c \vert \textbf{z}, \pmb{\theta}_c) $$ : likelihood distribution
```